js/web/test/unittests/backends/webgl/test-conv-utils.ts

```diff
@@ -1,14 +1,18 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+import {Float16Array} from '@petamoriken/float16';
```

Comment:
no need to add this as we don't plan to add f16 support for webgl

Reply:
done